http-client/src/main/java/io/micronaut/http/client/netty/DefaultHttpClient.java

```diff
@@ -1475,229 +1170,6 @@ protected Object getLoadBalancerDiscriminator() {
         return null;
     }
 
-    private void initBootstrapForProxy(Bootstrap bootstrap, boolean ssl, String host, int port) {
-        Proxy proxy = configuration.resolveProxy(ssl, host, port);
-        if (proxy.type() != Type.DIRECT) {
-            bootstrap.resolver(NoopAddressResolverGroup.INSTANCE);
-        }
-    }
-
-    /**
-     * Creates an initial connection to the given remote host.
-     *
-     * @param request         The request
-     * @param uri             The URI to connect to
-     * @param sslCtx          The SslContext instance
-     * @param isStream        Is the connection a stream connection
-     * @param contextConsumer The logic to run once the channel is configured correctly
-     * @return A ChannelFuture
-     * @throws HttpClientException If the URI is invalid
-     */
-    protected ChannelFuture doConnect(
```

Comment:
Whilst this is an internal class and this change is probably fine we will need to review whether downstream subclasses use any of these deleted protected methods

Reply:
The ribbon module from the Netflix module used to subclass these but that is deprecated and archived so maybe this is not a concern anymore